crates/auto_update/src/auto_update.rs

```diff
@@ -575,18 +577,20 @@ async fn download_remote_server_binary(
     cx: &AsyncAppContext,
 ) -> Result<()> {
     let mut target_file = File::create(&target_path).await?;
-    let (installation_id, release_channel, telemetry) = cx.update(|cx| {
+    let (installation_id, release_channel, telemetry_enabled) = cx.update(|cx| {
         let installation_id = Client::global(cx).telemetry().installation_id();
         let release_channel =
             ReleaseChannel::try_global(cx).map(|release_channel| release_channel.display_name());
-        let telemetry = TelemetrySettings::get_global(cx).metrics;
+        let telemetry_enabled = TelemetrySettings::get_global(cx).metrics;
 
-        (installation_id, release_channel, telemetry)
+        (installation_id, release_channel, telemetry_enabled)
     })?;
     let request_body = AsyncBody::from(serde_json::to_string(&UpdateRequestBody {
         installation_id,
         release_channel,
-        telemetry,
+        telemetry: telemetry_enabled,
+        is_staff: None,
```

Comment:
@ConradIrwin, how does the remote install work? Is it initiated from the locally installed version, where I actually could have the is_staff information to pass up?